mesonbuild/scripts/depfixer.py

```diff
@@ -154,7 +154,7 @@ def open_bf(self, bfile: str) -> None:
     def close_bf(self) -> None:
         if self.bf is not None:
             if self.bf_perms is not None:
-                os.fchmod(self.bf.fileno(), self.bf_perms)
+                os.chmod(self.bf.fileno(), self.bf_perms)
```

Comment:
`os.chmod` on Windows does not work when passing a file descriptor to it like is done here

Reply:
This function is only called from posix platform. Python doc says that since Python 3.3, chmod is equivalent to fchmod. Changing this is a quick fix to make mypy happy, even on Windows...